### ISSUE TITLE
Render Markdown (`type: markdown`)

### ISSUE THREAD

**Issue by borekb** [sindresorhus/linkify-issues#8]
Would be nice if `#123` could produce `[#123](https://github.com/owner/repo/issues/123)`.

**Comment by sindresorhus** [comment]
Yeah, I guess we could add `markdown` as a value in the `type` option. However, we need to note that the `attributes` option will be a noop then.
